File: WorldFeed/Server/src/WorldFeed.Common/Events/UnexpectedExceptionThrownEvent.cs
namespace WorldFeed.Common.Events
{
    using System;

    /// <summary>
    /// An exception that could not be handled by Tweetinvi was thrown
    /// Please report such errors on github
    /// </summary>
    public class UnexpectedExceptionThrownEvent
    {
        public UnexpectedExceptionThrownEvent(Exception exception)
        {
            this.Exception = exception;
        }

        public Exception Exception { get; }
    }
}

File: WorldFeed/Client/src/app/core/Public/Events/UnexpectedExceptionThrownEvent.ts
import Exception from "../../../c#-objects/TypeScript.NET-Core/packages/Core/source/Exception";

// An exception that could not be handled by WorldFeed was thrown. Please report such errors on github
export class UnexpectedExceptionThrownEvent {

  constructor(exception: Exception) {
    this.Exception = exception;
  }

  public Exception: Exception;
}
